src/otx/algo/modules/conv_module.py

```diff
@@ -0,0 +1,357 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) OpenMMLab. All rights reserved.
+
+"""This implementation copied ConvModule of mmcv.cnn.bricks.ConvModule."""
+# TODO(someone): Revisit mypy errors after deprecation of mmlab
+# mypy: ignore-errors
+from __future__ import annotations
+
+import warnings
+from functools import partial
+from typing import TYPE_CHECKING
+
+import torch
+from torch import nn
+from torch.nn.modules.batchnorm import _BatchNorm as BatchNorm
+from torch.nn.modules.instancenorm import _InstanceNorm as InstanceNorm
+
+from otx.algo.utils.mmengine_utils import constant_init, kaiming_init
+
+from .activation import build_activation_layer
+from .conv import build_conv_layer
+from .norm import build_norm_layer
+from .padding import build_padding_layer
+
+if TYPE_CHECKING:
+    from torch.nn.modules.conv import _ConvNd as ConvNd
+
+
+def efficient_conv_bn_eval_forward(bn: BatchNorm, conv: ConvNd, x: torch.Tensor) -> torch.Tensor:
```

Comment:
I think it is for efficientbet-b0, right?

Reply:
Since this is a module that is used across the board regardless of task, I just imported the code from that module file as-is. I'll check it out later and remove it if it's unnecessary.